lib/cldr.js

```diff
@@ -2091,6 +2091,43 @@ Cldr.prototype = {
 
     return weekData;
   },
+
+  extractTTSCharacterLabels(localeId) {
+    this.checkValidLocaleId(localeId);
+    const finder = this.createFinder(
+      this.getPrioritizedDocumentsForLocale(localeId, 'annotations')
+    );
+
+    const characterLabels = {};
+    finder('/ldml/annotations/annotation[@type="tts"]').forEach((node) => {
+      characterLabels[node.getAttribute('cp')] =
+        characterLabels[node.getAttribute('cp')] ?? node.textContent;
```

Comment:
This package still supports node.js 10 and 12, which don't support nullish coalescing operator. I know that CI is broken right now -- I haven't had time to switch to Github Actions -- but could you try to avoid it?